src/Pyz/Yves/MultiFactorAuth/Theme/default/views/user-set-up-page/set-up-page.twig

```diff
@@ -0,0 +1,79 @@
+{% extends template('page-layout-agent', 'AgentPage') %}
```

Comment:
It works for now but it have to be fixed later in https://spryker.atlassian.net/browse/FRW-10751